diff --git a/src/ui/components/ytMobile05/index.jsx b/src/ui/components/ytMobile05/index.jsx
index 77c1981..a00c8fd 100644
--- a/src/ui/components/ytMobile05/index.jsx
+++ b/src/ui/components/ytMobile05/index.jsx
@@ -679,9 +679,9 @@ const ComponentVideo = () => {
 
   return (
     <>
-      <VideoContainer className="video-Container" >
+      <VideoContainer className={`${stateActive === true ? "activeProgress" : "disable"}`} >
         {/* <Video ref={videoRef} muted opacity={opacity} onProgress={() => videoProgress()} onClick={() => play()} src={playlistMain} className={`video`} controlsList="nodownload" preload="auto" type="video/mp4" /> */}
-        <Video ref={videoRef} muted onClick={() => opacityPlay()} onProgress={() => videoProgress()} src={playlistMain} className={`video`} controlsList="nodownload" type="video/mp4" />
+        <Video ref={videoRef} muted onTouchStart={() => opacityPlay()} onProgress={() => videoProgress()} src={playlistMain} className={`video`} controlsList="nodownload" type="video/mp4" />
         {isLoading === true ? (
           <Spinner>
             <div className="spinner"></div>
diff --git a/src/ui/components/ytMobile05/styles.js b/src/ui/components/ytMobile05/styles.js
index 075203c..6c09326 100644
--- a/src/ui/components/ytMobile05/styles.js
+++ b/src/ui/components/ytMobile05/styles.js
@@ -10,6 +10,8 @@ export const VideoContainer = styled.div`
     flex-direction: column;
     align-items: center;
     /* justify-content: center; */
+    z-index: 1000;
+    
 
     /* Remover realce azul no toque (Mobile e WebKit) */
     -webkit-tap-highlight-color: transparent;
@@ -24,6 +26,17 @@ export const VideoContainer = styled.div`
       opacity: 0;
     }
 
+    &.activeProgress::before {
+    content: "";
+    position: absolute;
+    top: 0;
+    left: 0;
+    width: 100%;
+    height: 100%;
+    background: rgba(0, 0, 0, 0.3); /* Fundo escuro com opacidade */
+    transition-duration: 0.5s;
+  }
+
     /* @media (min-width: 768px) {
   .controlsContainer, .progress-bar {
     display: none;
