diff --git a/src/assets/face/face01.jpg b/src/assets/face/face01.jpg
deleted file mode 100644
index 426dc72..0000000
Binary files a/src/assets/face/face01.jpg and /dev/null differ
diff --git a/src/assets/face/face02.jpg b/src/assets/face/face02.jpg
deleted file mode 100644
index 78a4ff3..0000000
Binary files a/src/assets/face/face02.jpg and /dev/null differ
diff --git a/src/assets/face/face03.jpg b/src/assets/face/face03.jpg
deleted file mode 100644
index 24a92df..0000000
Binary files a/src/assets/face/face03.jpg and /dev/null differ
diff --git a/src/assets/face/face04.jpg b/src/assets/face/face04.jpg
deleted file mode 100644
index b4a2f73..0000000
Binary files a/src/assets/face/face04.jpg and /dev/null differ
diff --git a/src/pages/frontend/degrade/index.jsx b/src/pages/frontend/degrade/index.jsx
index 52a1420..d7ec5ce 100644
--- a/src/pages/frontend/degrade/index.jsx
+++ b/src/pages/frontend/degrade/index.jsx
@@ -1,10 +1,10 @@
 import styled from "styled-components";
 import { FaRegEnvelope } from "react-icons/fa"
 
-import face01 from "../../../assets/face/face01.jpg"
-import face02 from "../../../assets/face/face02.jpg"
-import face03 from "../../../assets/face/face03.jpg"
-import face04 from "../../../assets/face/face04.jpg"
+import face01 from "../../../assets/face/face01.png"
+import face02 from "../../../assets/face/face02.png"
+import face03 from "../../../assets/face/face03.png"
+import face04 from "../../../assets/face/face04.png"
 import React from "react";
 
 // sem efeito   ### 
@@ -51,7 +51,7 @@ border: solid 1px #000000;
 overflow: hidden;
 `;
 
-const DivTransparent02= styled.div`
+const DivTransparent02 = styled.div`
   position: relative;
 
   &::after, &::before {
@@ -96,53 +96,53 @@ animation: moveBottom 5s linear  infinite;
 // ==========  ==========
 
 const Degrade = () => {
-    const list = [
-        { id: 1, img: face01, nome: "01" },
-        { id: 2, img: face02, nome: "02" },
-        { id: 3, img: face03, nome: "03" },
-        { id: 4, img: face04, nome: "04" },
-    ]
-    return (
-        <div style={{ display: "flex", flexDirection: "column", height: "100vh" }}>
-            <h1>DivTransparent01</h1>
-            <ContainerNoEffect>
-                <DivTransparent01>
-                    <ul>
-                        {list.map((el, i) => {
-                            return (
-                                <React.Fragment key={i}>
-                                    <li style={{ display: "flex", alignItems: "center" }}>
-                                        <Img src={el.img} />
-                                        <span>{el.nome}</span>
-                                        <span><FaRegEnvelope /></span>
-                                    </li>
-                                </React.Fragment>
-                            )
-                        })}
-                    </ul>
-                </DivTransparent01>
-            </ContainerNoEffect><br /><br /><br /><br />
-
-            <h1>DivTransparent02</h1>
-            <ContainerIndeed>
-                <DivTransparent02>
-                    <ul>
-                        {list.map((el, i) => {
-                            return (
-                                <React.Fragment key={i}>
-                                    <Li style={{ display: "flex", alignItems: "center" }}>
-                                        <Img01 src={el.img} />
-                                        <span>{el.nome}</span>
-                                        <span><FaRegEnvelope /></span>
-                                    </Li>
-                                </React.Fragment>
-                            )
-                        })}
-                    </ul>
-                </DivTransparent02>
-            </ContainerIndeed>
-        </div>
-    )
+  const list = [
+    { id: 1, img: face01, nome: "01" },
+    { id: 2, img: face02, nome: "02" },
+    { id: 3, img: face03, nome: "03" },
+    { id: 4, img: face04, nome: "04" },
+  ]
+  return (
+    <div style={{ display: "flex", flexDirection: "column", height: "100vh" }}>
+      <h1>DivTransparent01</h1>
+      <ContainerNoEffect>
+        <DivTransparent01>
+          <ul>
+            {list.map((el, i) => {
+              return (
+                <React.Fragment key={i}>
+                  <li style={{ display: "flex", alignItems: "center" }}>
+                    <Img src={el.img} />
+                    <span>{el.nome}</span>
+                    <span><FaRegEnvelope /></span>
+                  </li>
+                </React.Fragment>
+              )
+            })}
+          </ul>
+        </DivTransparent01>
+      </ContainerNoEffect><br /><br /><br /><br />
+
+      <h1>DivTransparent02</h1>
+      <ContainerIndeed>
+        <DivTransparent02>
+          <ul>
+            {list.map((el, i) => {
+              return (
+                <React.Fragment key={i}>
+                  <Li style={{ display: "flex", alignItems: "center" }}>
+                    <Img01 src={el.img} />
+                    <span>{el.nome}</span>
+                    <span><FaRegEnvelope /></span>
+                  </Li>
+                </React.Fragment>
+              )
+            })}
+          </ul>
+        </DivTransparent02>
+      </ContainerIndeed>
+    </div>
+  )
 }
 
 export default Degrade
\ No newline at end of file
